pydoctor/model.py

```diff
@@ -404,6 +406,34 @@ def _localNameToFullName(self, name: str) -> str:
     def module(self) -> 'Module':
         return self
 
+    @property
+    def docformat(self) -> Optional[str]:
+        """Value of the the C{__docformat__} variable of this module.
+
+        C{pydoctor} will try to parse all module's docstrings with the 
+        specified parser. Language is ignored and parser name is lowercased.
+        
+        This is all valid::
+
+            __docformat__ = "reStructuredText en"
+            __docformat__ = "epytext"
+            __docformat__ = "restructuredtext"
+
+        If no C{__docformat__} variable was found in the module, or its
+        contents could not be parsed, this is L{None}.
+        """
+        if self._docformat:
+            return self._docformat
+        elif self.parent:
+            _docformat = getattr(self.parent, "docformat", None)
+            if _docformat:
+                assert isinstance(_docformat, str)
+                return _docformat
```

Comment:
It is handling the docformat inheritence from the package if any.

Reply:
I think it would be better to assert that the parent is a `Module` instead of using `getattr` + `assert` on the fetched value.

Or have `parent: Optional['Package']` as an annotation on `Module`, if mypy allows that.